src/Docker.DotNet/Endpoints/IContainerOperations.cs

```diff
@@ -40,6 +41,8 @@ public interface IContainerOperations
 
         Task<Stream> GetContainerLogsAsync(string id, ContainerLogsParameters parameters, CancellationToken cancellationToken);
```

Comment:
Ya now I think I mislead you. We only expose the Interfaces externally. I think you need to make the [Obsolete] tags here instead. My bad